.github/workflows/versioning.yml

```diff
@@ -11,6 +11,7 @@ jobs:
   versioning:
     name: Versioning
     runs-on: ubuntu-latest
+    if: github.repository == 'mauroalderete/action-assign-labels'
```

Comment:
I've noticed a small typo in one of the commits. Once we start the review process, it would be great if we could address that as well.